dev-java/java-service-wrapper/java-service-wrapper-3.5.50.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+inherit java-pkg-2 java-ant-2 toolchain-funcs
+
+MY_PN="wrapper"
+MY_P="${MY_PN}_${PV}_src"
+DESCRIPTION="A wrapper that makes it possible to install a Java Application as daemon"
+HOMEPAGE="https://wrapper.tanukisoftware.com/"
+SRC_URI="https://${MY_PN}.tanukisoftware.com/download/${PV}/${MY_P}.tar.gz"
+
+LICENSE="tanuki-community"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+RDEPEND="
+	virtual/jre:1.8"
+DEPEND="
+	virtual/jdk:1.8
```

Comment:
>=virtual/jdk-1.8:*

If this runs into errors using openjdk-17 you should switch to `java-pkg-simple.eclass`

Reply:
Switched to this eclass.  Still need a couple things from `java-pkg-2.eclass` during the install phase so inheriting both.  No more ant though.